config.py

```diff
@@ -0,0 +1,74 @@
+#!flask/bin/python/
+"""
+Configuration settings for the Bucket List API
+The definition of the different configuration settings:
+- Development Configuration
+- Testing Configuration
+- Production Configuration.
+"""
+
+import os
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config:
+    """
+    The definition of the global configuration is defined here.
+    """
+
+    SECRET_KEY = b'\xea&#Tb\xb0\x04\x12\x06c+r$\xffjQa\x0cg0'
+    SQLALCHEMY_DATABASE_URI = 'sqlite:///' + os.path.join(basedir, 'bucket.db')
+    SQLALCHEMY_COMMIT_ON_TEARDOWN = True
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+    DEBUG = False
+
+    @staticmethod
+    def init_app(app):
+        """
+        This static method initializes the application using whatever
```

Comment:
Nope, it doesn't..

`app.config.from_object` correctly handles this as you have done [here](https://github.com/andela-odurodola/BucketListAPI/blob/develop/app/__init__.py#L15)

The redundant `init_app` staticmethod is not needed at all and should be taken out.
Remember to remove the call to the method in your `BucketListAPI/app/__init__.py` file as well
As you can see, all the method does is `pass`.

That alone should be enough to give you a hint that it's just an unneeded interface